<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="uk.ac.sheffield.com1003.cafe.TestRecipeEqualsNoIngredients" tests="1" skipped="0" failures="1" errors="0" timestamp="2024-03-18T23:00:32" hostname="Ruizhens-MacBook-Pro.local" time="0.0">
  <properties/>
  <testcase name="testRecipeEqualsNoIngredients" classname="uk.ac.sheffield.com1003.cafe.TestRecipeEqualsNoIngredients" time="0.0">
    <failure message="java.lang.NullPointerException" type="java.lang.NullPointerException">java.lang.NullPointerException
	at uk.ac.sheffield.com1003.cafe.Recipe.isReady(Recipe.java:65)
	at uk.ac.sheffield.com1003.cafe.Recipe.equals(Recipe.java:78)
	at org.junit.Assert.isEquals(Assert.java:133)
	at org.junit.Assert.equalsRegardingNull(Assert.java:129)
	at org.junit.Assert.assertEquals(Assert.java:112)
	at org.junit.Assert.assertEquals(Assert.java:146)
	at uk.ac.sheffield.com1003.cafe.TestRecipeEqualsNoIngredients.testRecipeEqualsNoIngredients(TestRecipeEqualsNoIngredients.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:750)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
